src/UmbracoFileSystemProviders.Azure/AzureBlobFileSystem.cs

```diff
@@ -115,12 +120,14 @@ public AzureBlobFileSystem(string alias)
                     useDefaultRoute = "true";
                 }
 
-                string accessType = ConfigurationManager.AppSettings[$"{UseDefaultRootKey}:{alias}"];
-                if (string.IsNullOrWhiteSpace(useDefaultRoute))
+                string accessType = ConfigurationManager.AppSettings[$"{UsePrivateContainerKey}:{alias}"];
+                if (string.IsNullOrWhiteSpace(accessType))
                 {
-                    useDefaultRoute = "true";
+                    accessType = "true";
                 }
 
+                this.FileSystem = AzureFileSystem.GetInstance(containerName, rootUrl, connectionString, maxDays, useDefaultRoute, accessType);
```

Comment:
You've got the same line of code in here twice. I'll remove it though once I've merged the PR.